config/client.lua

```diff
@@ -163,7 +163,7 @@ return {
                     vec3(-716.94, -1326.88, 0)
                 },
                 size = vec3(8, 8, 6),
-                targetDistance = 5,
+                targetDistance = 10,
```

Comment:
this seems long, no?

Reply:
For the smaller vehicles (like jetskis for examples) it might be hard to target them properly from the decks, since you want the spawn point to have enough space around it for the larger models.